src/utils/dates.js

```diff
@@ -0,0 +1,132 @@
+import moment from 'moment';
```

Comment:
I hate this file and everything it represents. I have been wanting to implement this alternative for a while now: https://date-fns.org/. Gets a good rep too.

Reply:
I am aware that I wrote most of this file for those reading the above comment I thinking I am being mean.

Reply:
Maybe we can set this as an improvement to be done

Reply:
Agree.